docs/installation.md

```diff
@@ -1,68 +1,68 @@
-# Install Guide
+# Installation guide
 
 ## Prerequisites
-See [here](user-guide.md#before-you-begin) for install requirements
+See the [user guide](user-guide.md#before-you-begin) for install requirements
 
-## Install Options
-The Liberty Tools feature can be installed as new software either by:
+## Installation options
+You can install Liberty Tools as new software by one of the following methods:
 
-*  using the Eclipse Marketplace, OR
-*  configuring an update site, OR
-*  downloading the artifacts and installing using the archive
+*  Using the Eclipse Marketplace
+*  Configuring an update site
+*  Downloading the artifacts and installing them by using the archive
 
 ## Using Eclipse Marketplace
 
 1. Point your browser to the Liberty Tools marketplace entry: https://marketplace.eclipse.org/content/liberty-tools
-2. Drag the "Install" button to the toolbar of your Eclipse IDE
+2. Drag the **Install** button to the toolbar of your Eclipse IDE
 
-## Using Eclipse Marketplace client
+## Using the Eclipse Marketplace client
 
-1. Go to menu:  `Help`->`Eclipse Marketplace` and type **"Liberty Tools"**.
+1. From the Eclipse menu in your IDE, select **Help** > **Eclipse Marketplace** and type **"Liberty Tools"**.
 
     **NOTE:** Do NOT select the "IBM Liberty Developer Tools" selection, which is an earlier, different set of IDE features/plugins, though with the similar name.
 
 ## Using Help -> Install New Software
 
-The Liberty Tools feature can be installed using the artifacts provided in the form of an update site or through an archive.
+You can use the **Install New Software** option in the Eclipse **Help** menu to install Liberty Tools by using an update site or through an archive.
 
-1. Go to menu: `Help`->`Install New Software`.
+1. From the Eclipse menu in your IDE, select **Help** > **Install New Software**.
```

Comment:
I think "menu bar" is a better fit than "Eclipse menu" so I changed it in my commit.